logger/logger.go

```diff
@@ -46,6 +47,7 @@ func init() {
 	AppLog = log.WithFields(logrus.Fields{"component": "SMF", "category": "App"})
 	InitLog = log.WithFields(logrus.Fields{"component": "SMF", "category": "Init"})
 	CfgLog = log.WithFields(logrus.Fields{"component": "SMF", "category": "CFG"})
+	DataRepoLog = log.WithFields(logrus.Fields{"component": "AMF", "category": "DRepo"})
```

Comment:
I think this should be `SMF` correct?
```suggestion
	DataRepoLog = log.WithFields(logrus.Fields{"component": "SMF", "category": "DRepo"})
```